README.md

```diff
@@ -858,6 +858,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [Better Stack](https://betterstack.com/better-uptime) - Uptime monitoring, incident management, on-call scheduling/alerting, and status pages in a single product. The free plan includes ten monitors with 3-minute check frequency and status pages.
   * [Pulsetic](https://pulsetic.com) - 10 monitors, 6 Months of historical Uptime/Logs, unlimited status pages, and custom domains included! For infinite time and unlimited email alerts for free. You don't need a credit card.
   * [Wachete](https://www.wachete.com) - monitor five pages, checks every 24 hours.
+  * [WebGazer](https://www.webgazer.io/for/open-source) - 50 uptime monitors, 10 heartbeat monitors and status pages free. 
```

Comment:
We've had these kind of "apply" based workflows in the past for companies offering services for open source and have denied them.  

The requirements of what kind of projects is accepted etc is opaque and its a extra hurdle that probably means its not worth it. 

Good flows is a github signup flow that checks the project is open source or not by its license, thats it.  If more checks are needed then no its not "for open source" it is for the projects the company choose to pick.

So I'll close this one on the basis of having not accepted similar ones in the past.

thanks